.github/workflows/deploy-preview.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   deploy-preview:
     runs-on: ubuntu-latest
-    if: ${{ github.repository_owner == 'fermyon' }}
```

Comment:
Could you expand on why the existing check would not be enough?

Specifically if this is something we'd want to document to users of the action.

Reply:
@vdice I think the goal is more more "do not run preview workflow on PRs originating from forks".  From what I can tell, the issue that we're seeing is that preview seems to fail on _PRs created from forks_ (cf. https://github.com/fermyon/developer/actions/workflows/deploy-preview.yml - Tim's PRs from this repo succeed, mine and Kate's and Rajat's from our forks fail).  These PRs are in an owned-by-Fermyon repo and so pass this check; but I guess they don't have access to the secret, so the deployment fails.

Reply:
Ah ha, thank you @itowlson.  TIL.  Indeed, I've just seen the same behavior on my PR for a different item, from my fork.